testing/execution.ts

```diff
@@ -586,6 +586,7 @@ export async function executeFormulaOrSyncWithRawParams<T extends FormulaSpecifi
  * of state or global variables between iterations.
  *
  * For now, use `coda execute --vm` to simulate that level of isolation.
+ * @deprecated Use {@link executeSyncFormula} instead.
  */
 export async function executeSyncFormulaFromPackDef<T extends object = any>(
```

Comment:
shouldn't this function just call the new one and return a subset of the result?

Reply:
HA yes